amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -89,7 +89,7 @@ class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
   render() {
     const subTextClass = `subtext ${this.state.optionalSubTextClass}`;
     return (
-      <div id="search-bar" className="col-xs-12 col-md-offset-1 col-md-10">
```

Comment:
Curious what the result of removing these styles were, is the layout still consistent?

Reply:
These classes were moved to the parent element in Search Page line 254 so the layout remains the same.